src/compile/compile.ts

```diff
@@ -35,8 +35,12 @@ export function compile(spec) {
     keys(config.scene).length > 0 ? scene(config) : {},
     {
       data: compileData(model).concat([compileLayoutData(model)]),
-      marks: [compileRootGroup(model)]
-    });
+      marks: [].concat(
+       (spec.title ? [compileTitle(model)] : []),
```

Comment:
For all `spec.title` reference, could you please add `model.title()` and call it?   (We plan to get rid of `model.spec()` so we don't want to call it in too many more places)